src/agreement.rs

```diff
@@ -178,25 +178,12 @@ impl PublicKey {
 }
 
 /// An unparsed, possibly malformed, public key for key agreement.
-pub struct UnparsedPublicKey<B: AsRef<[u8]>> {
+#[derive(Clone, Copy)]
+pub struct UnparsedPublicKey<B> {
```

Comment:
Regarding the code coverage for `UnparsedPublicKey`, I guess the explicit implementations were also untested, so I'm going to call that out of scope.

Reply:
Would a follow-up issue be best to cover adding these?

Reply:
If it were anything other than "missing code coverage," probably. Right now we can use the code coverage reports to see what code isn't hit by code coverage yet. Currently about 7%, although that's an over-estimate because there's some testing done not in CI that isn't counted towards the measurement yet.